src/store/entities/constants.ts

```diff
@@ -1 +1,28 @@
 export const SAVE_NORMALIZED_ENTITIES = 'SAVE_NORMALIZED_ENTITIES';
+
+export const TYPE_DOMAIN_MAPPING = {
+  contact: 'contacts',
+  company: 'companies',
+  user: 'users',
+  businessType: 'businessTypes',
+  deal: 'deals',
+  dealPhase: 'dealPhases',
+  dealSource: 'dealSources',
+  quotation: 'quotations',
+  event: 'events',
+  activityType: 'activityTypes',
+  invoice: 'invoices',
+  creditNote: 'creditNotes',
+  taxRate: 'taxRates',
+  paymentTerm: 'paymentTerms',
+  product: 'products',
+  productCategory: 'productCategories',
+  project: 'projects',
+  milestone: 'milestones',
+  timeTracking: 'timeTracking',
+  departments: 'departments',
+  customFieldDefinition: 'customFieldDefinitions',
+  workType: 'workTypes',
+  task: 'tasks',
+  todo: 'tasks.list', // Deprecated, `task` should be used instead
```

Comment:
tasks.list? Is this supposed to be like this?

Reply:
Whoops, that's a typo indeed